rest/config_database.go

```diff
@@ -153,17 +153,19 @@ func DefaultDbConfig(sc *StartupConfig, useXattrs bool) *DbConfig {
 				ExpirySeconds:        base.IntPtr(int(db.DefaultChannelCacheAge.Seconds())),
 			},
 		},
-		StartOffline:                base.BoolPtr(false),
-		OIDCConfig:                  nil,
-		OldRevExpirySeconds:         base.Uint32Ptr(base.DefaultOldRevExpirySeconds),
-		ViewQueryTimeoutSecs:        base.Uint32Ptr(base.DefaultViewTimeoutSecs),
-		LocalDocExpirySecs:          base.Uint32Ptr(base.DefaultLocalDocExpirySecs),
-		EnableXattrs:                base.BoolPtr(base.DefaultUseXattrs),
-		SecureCookieOverride:        base.BoolPtr(sc.API.HTTPS.TLSCertPath != ""),
-		SessionCookieName:           "",
-		SessionCookieHTTPOnly:       base.BoolPtr(false),
-		AllowConflicts:              base.BoolPtr(base.DefaultAllowConflicts),
-		NumIndexReplicas:            base.UintPtr(DefaultNumIndexReplicas),
+		StartOffline:          base.BoolPtr(false),
+		OIDCConfig:            nil,
+		OldRevExpirySeconds:   base.Uint32Ptr(base.DefaultOldRevExpirySeconds),
+		ViewQueryTimeoutSecs:  base.Uint32Ptr(base.DefaultViewTimeoutSecs),
+		LocalDocExpirySecs:    base.Uint32Ptr(base.DefaultLocalDocExpirySecs),
+		EnableXattrs:          base.BoolPtr(base.DefaultUseXattrs),
+		SecureCookieOverride:  base.BoolPtr(sc.API.HTTPS.TLSCertPath != ""),
+		SessionCookieName:     "",
+		SessionCookieHTTPOnly: base.BoolPtr(false),
+		AllowConflicts:        base.BoolPtr(base.DefaultAllowConflicts),
+		Index: &IndexConfig{
+			NumReplicas: base.UintPtr(DefaultNumIndexReplicas),
```

Comment:
This doesn't change anything other than displaying "1" as default when `include_runtime=true` is set on a GET config request but I think we should do it.

```suggestion
			NumReplicas:   base.UintPtr(DefaultNumIndexReplicas),
			NumPartitions: base.Ptr(uint32(db.DefaultNumIndexPartitions),
```

Reply:
I think this is fine but let me run through jenkins before merging.